### DISCUSSION TITLE
Overriding of existing country seems not to work

### DISCUSSION THREAD

**Discussion by codeDealing** [focus-shift/jollyday#496] [Q&A] [answered]
Hey there!

I have tried as described in the main page to override some holidays for Germany.

The new XML definition was added under classpath (resources - i am using Spring boot). It seems that the classpath loader is not using the provided custom definition. Am i missing something?

Thanks in advance

**Comment by derTobsch** [comment, marked as answer]
Hey @codeDealing,
we use it and it does work. You can take a look here https://github.com/urlaubsverwaltung/urlaubsverwaltung/tree/main/src/main/resources

**Reply by codeDealing** [reply]
Thank you for the quick response :) !!!

**Reply by derTobsch** [reply]
@codeDealing did you get your mistake and can share it?

**Comment by MichelAlBoustany** [comment]
I am also facing the same issue and it also doesnt work unfortunately

**Reply by derTobsch** [reply]
can you provide an example?
